README.md

```diff
@@ -284,12 +284,13 @@ auth token.
 
 ## `pogobuf.Utils` methods
 ### `getCellIDs(lat, lng)` ⇒ <code>array</code> *(static)*
-Provides cell IDs of nearby cells based on the given coords.
+Provides cell IDs of nearby cells based on the given coords and radius.
 
 | Param | Type | Description |
 | --- | --- | --- |
 | lat | <code>number</code> | Latitude |
 | lng | <code>number</code> | Longitude |
+| radius | <code>number</code> | Radius of the square |
```

Comment:
That would be great to mention in the docs, as I'll bet people will assume kilometers.